packages/react-internal/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -188,14 +188,15 @@ export class CommandBarBase extends React.Component<ICommandBarProps, {}> implem
     // The prop types between these three possible implementations overlap enough that a force-cast is safe.
     let Type = DefaultButtonAs;
 
-    if (CommandBarButtonAs) {
-      Type = composeComponentAs(CommandBarButtonAs, Type);
-    }
-
     if (ButtonAs) {
       Type = composeComponentAs(ButtonAs, Type);
     }
 
```

Comment:
You could actually leave it as is and just do `else if (ButtonAs)` so that `composeComponentAs` is only run once.